python/lsst/validate/drp/validate.py

```diff
@@ -289,6 +290,8 @@ def runOneFilter(repo, visitDataIds, metrics, brightSnr=100,
         PsfShape measurements).
     verbose : bool, optional
         Output additional information on the analysis steps.
+    skipGalaxies : bool, optional
+        Whether to skip processing galaxies even if model measurements are available; default False.
```

Comment:
I don't think it's necessary to support this option.  If we can calculate them we should.

Reply:
I think we can since there should always be a `slot_modelFlux`, but if CModel hasn't been run then that defaults to `base_GaussianFlux`, which isn't especially useful.